resources/providers/gcplib/inventory/provider.go

```diff
@@ -460,11 +493,23 @@ func getEcsGcpCloudData(ctx context.Context, crm *ResourceManagerWrapper, keys G
 
 func getOrganizationId(ancestors []string) string {
 	last := ancestors[len(ancestors)-1]
-	return strings.Split(last, "/")[1]
+	parts := strings.Split(last, "/") // organizations/1234567890
+
+	if parts[0] == "organizations" {
+		return parts[1]
+	}
+
+	return ""
 }
 
 func getProjectId(ancestors []string) string {
-	return strings.Split(ancestors[0], "/")[1]
+	parts := strings.Split(ancestors[0], "/") // projects/1234567890
+
+	if parts[0] == "projects" {
+		return parts[1]
+	}
+
+	return ""
 }
```

Comment:
updated in order to be aligned with resources that are being created at the folder/org level.
Their ancestors will look like `[folders/123, org/123]`